### DISCUSSION TITLE
sthemer dependency update

### DISCUSSION THREAD

**Discussion by sjaghori** [ivanhofer/sthemer#1] [General]
Hi Ivan,

I was trying to update the project to the released version of sveltekit, but I cannot because all of the dependencies including sthemer must be on the current version (vite4...).

Do you plan to update the dependencies of this library soon?

**Comment by ivanhofer** [comment]
Hi @sjaghori,
what error do you get?

**Reply by sjaghori** [reply]
This kind of error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: rb-my-experience@0.0.1
npm ERR! Found: @sveltejs/kit@1.0.0-next.514
npm ERR! node_modules/@sveltejs/kit
npm ERR!   peer @sveltejs/kit@"^1.0.0-next" from sthemer@1.2.0
npm ERR!   node_modules/sthemer
npm ERR!   dev @sveltejs/kit@"1.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! dev @sveltejs/kit@"1.0.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: vite@4.0.3
npm ERR! node_modules/vite
npm ERR!   peer vite@"^4.0.0" from @sveltejs/kit@1.0.0
npm ERR!   node_modules/@sveltejs/kit
npm ERR!     dev @sveltejs/kit@"1.0.0" from the root project
```

But I think that is npm not handling the peer dependencies correctly.
Removing node_modules didn't help, but removing package.lock.json (not the best idea) and running npm install helped.

**Reply by ivanhofer** [reply]
I don't think it is an issue with `sthemer`. You probably need to install `vite` 4 first and then update all other dependencies.
Anyways. I have updated all dependencies for `sthemer` and released a new version. Please let me know if everything works.
By the way, you can get rid of the `sthemer` config options in your `vite.config.ts`.

**Reply by sjaghori** [reply]
awesome thank you!
